### ISSUE TITLE
Concatenate multiple environment files

### ISSUE THREAD

**Issue by tdegeus** [mamba-org/setup-micromamba#154]
Is there a way to concatenate multiple environment files? 

I have two libraries that I want to compile in the CI, both with their own dependencies, both listed in their respective environment files. I do have https://github.com/tdegeus/conda_envfile for that, but that already requires python and 
https://github.com/tdegeus/conda_envfile/blob/372dc49138c8f2c7c6783c05a791f7d22de32662/pyproject.toml#L7C16-L7C17

**Comment by pavelzw** [comment]
Multiple environment files are currently not supported by setup-micromamba and I don't think that we will add support for this since this is kind of niche.

You could install a separate environment in an extra step or use a matrix to spawn multiple runners.
See #51

**Comment by peanutfun** [comment]
The micromamba docs now explicitly refer to the option of adding multiple environment files to the `micromamba create` command, see https://mamba.readthedocs.io/en/latest/user_guide/micromamba.html#conda-yaml-spec-files :

> You can pass multiple YAML spec files by repeating the -f,--file argument.

It would be great if this option could be added to the GitHub action, because otherwise one needs additional build steps for the following spec files, and only the initial environment can be cached.

**Comment by pavelzw** [comment]
@peanutfun does something like this work?

```yml
- uses: mamba-org/setup-micromamba@v1
  with:
    name: my-env
    environment-file: environment.yml
    create-args: -f environment2.yml
```

**Comment by peanutfun** [comment]
@pavelzw Thank you! Your suggestions appears to result in a valid `micromamba create` command, but even when I explicitly request the latest version of micromamba, I get the error 
```
  critical libmamba Can only handle 1 yaml file!
  info     libmamba ****************** Backtrace End ********************
  Error: The process '/home/runner/micromamba-bin/micromamba' failed with exit code 1
```
I can replicate this error locally. It seems like the docs of micromamba are incorrect.

**Comment by pavelzw** [comment]
yeah this was just merged into micromamba not long ago, maybe the new version is just not released yet (or they wait for micromamba 2.0 to release this feature)

**Comment by peanutfun** [comment]
Thanks for the background info and the support! Then I'll be patient. I just raised an issue on the topic: https://github.com/mamba-org/mamba/issues/3196
